Sources/StreamChat/Extensions/URL+EnrichedURL.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+extension URL {
+    /// Enriches `URL` with `http` scheme if it's missing
+    var enrichedURL: URL {
+        guard scheme == nil else {
+            return self
+        }
+
+        return URL(string: "http://" + absoluteString) ?? self
```

Comment:
Should the preferred URL Scheme be `https` for security reasons?

Reply:
This was explained on slack. Leveraging http scheme by default is safe.